src/components/com_tjdashboard/site/assets/js/tjDashboardService.js

```diff
@@ -10,6 +10,7 @@ var TJDashboardService = {
 	api_dashboard_url : 'index.php?option=com_api&app=tjdashboard&resource=dashboard&id=',
 	api_widget_url : 'index.php?option=com_api&app=tjdashboard&resource=widget&id=',
 	get_renderers_url : 'administrator/index.php?option=com_tjdashboard&task=widget.getSupportedRenderers',
+	get_widget_params : 'administrator/index.php?option=com_tjdashboard&task=widget.getWidgetParams',
```

Comment:
Putting `administrator/` in the config URL seems a little odd to me. Should we do this or instead have 2 variables for base frontend url and base admin URL ?

@twsvaishali ?

Reply:
If this is getting called from frontend as well then this is wrong. We will need to move the task in frontend and use those urls only.

Reply:
@coolbung @twsvaishali @thite-amol 
This url used with "root_url" as defined globle variable which is for root path. So, I think this is not getting any wrong path

Reply:
@coolbung This will need more detailed discussion.

For now, I will merge this. The discussion will be pending for the component structure

Reply:
OK

Reply:
However can you propose the structure considering we may eventually have dashboard creation form in the frontend also.

Reply:
Will do this